tools/migration_testing/compare_responses.py

```diff
@@ -488,25 +146,41 @@ def send_requests(endpoint_infos, use_api_key):
 
 
 def main():
-  to_use = NEW_ENDPOINTS if USE_NEW_ENDPOINTS_ONLY else ENDPOINTS
+  if args.endpoints == "new":
+    endpoints = NEW_ENDPOINTS
+    error_tests = []
+  elif args.endpoints == "mixer":
+    endpoints = MIXER_ENDPOINTS
+    error_tests = MIXER_ERROR_TESTS
+  elif args.endpoints == "nl":
+    endpoints = NL_ENDPOINTS
+    error_tests = NL_ERROR_TESTS
+  else:
+    raise ValueError(
+        f"Invalid endpoint type. Use 'mixer', 'nl', or 'new'. Got: {args.endpoints}"
+    )
 
   print()
   print(f"{bcolors.BOLD}With API key{bcolors.ENDC}")
-  send_requests(to_use, use_api_key=True)
+  send_requests(endpoints, use_api_key=True)
 
   print()
   print(f"{bcolors.BOLD}Without API key{bcolors.ENDC}")
-  send_requests(to_use, use_api_key=False)
+  send_requests(endpoints, use_api_key=False)
 
-  if not USE_NEW_ENDPOINTS_ONLY:
+  if (len(error_tests) > 0):
     print()
     print(f"{bcolors.BOLD}Error tests{bcolors.ENDC}")
-    send_requests(ERROR_TESTS, use_api_key=True)
+    send_requests(error_tests, use_api_key=True)
 
 
 if __name__ == "__main__":
   parser = argparse.ArgumentParser(
       description="Compare API responses between two domains with an API key.")
+  parser.add_argument(
+      "endpoints",
+      help="Which set of endpoints to test. Values are 'mixer', 'nl', or 'new'."
```

Comment:
I feel this endpoints a bit confusing. maybe use the subdomain directly?

Reply:
Since the subdomain for Mixer API is just "api" I think that might also be confusing. I extended the comment - wdyt?

Reply:
Wonder if `endpoints`, `current_domain` and `new_domain` can be consolidated. But it's good as is.